pkg/eraser/eraser.go

```diff
@@ -195,11 +195,16 @@ func removeImages(c Client, targetImages []string) error {
 		}
 	}
 
-	// map for non-running imags by name
-	nonRunningNames := make(map[string]struct{}, len(allImages)-len(runningImages))
-	for key := range nonRunningImages {
-		if idMap[key] != nil && len(idMap[key]) > 0 {
-			nonRunningNames[idMap[key][0]] = struct{}{}
+	log.V(1).Info("List of non-running images by digest", "nonRunningImages", nonRunningImages)
```

Comment:
Probably should be debug only.

Reply:
See above, `log.V(1).Info` is a way to get debug-only logging here, as it raises the verbosity level by one. Since there's no `.Debug()` method, I wasn't sure how to go about that here.